example/app/src/main/java/com/rootstrap/androidcomposebase/ui/navigation/MainNav.kt

```diff
@@ -0,0 +1,29 @@
+package com.rootstrap.androidcomposebase.ui.navigation
+
+import androidx.compose.runtime.Composable
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import androidx.navigation.toRoute
+import com.rootstrap.androidcomposebase.ui.pages.login.LogInScreen
+import com.rootstrap.androidcomposebase.ui.pages.settings.SettingsScreen
+
+@Composable
+fun MainNavHost(isOSDarkTheme :Boolean, onThemeUpdated: () -> Unit) {
+
+    NavHost(
+        navController  = NavigationAppData.navController,
+        startDestination = Pages.LoginScreen
+    ){
+        composable<Pages.LoginScreen>{
+            LogInScreen()
+        }
+        composable<Pages.SettingsScreen>{
+            val args = it.toRoute<Pages.SettingsScreen>()
+            SettingsScreen(
+                email = args.email,
+                isOSDarkTheme = isOSDarkTheme,
+                onThemeUpdated = onThemeUpdated
+            )
+        }
+    }
+}
```

Comment:
I'll suggest to add a new line here, but maybe is more useful to add a linter that should be able to catch this before, what do you think? We can create a ticket to add it later